owncloudApp/src/main/res/values/strings.xml

```diff
@@ -727,20 +727,30 @@
-    <string name="release_notes_4_2_0_subtitle_4">The file list will be now refreshed automatically while uploading to the target folder</string>
-    <string name="release_notes_4_2_0_title_5">Download log files</string>
-    <string name="release_notes_4_2_0_subtitle_5">New option to download a log file to the Downloads folder of the device</string>
-    <string name="release_notes_4_2_0_title_6">Minor bugfixes</string>
-    <string name="release_notes_4_2_0_subtitle_6">Some bugs were fixed to improve experience in the app</string>
+    <string name="release_notes_4_2_0_title_1">Latest version with support for Android 6</string>
+    <string name="release_notes_4_2_0_subtitle_1">Current release will be the last one that supports Android 6</string>
+    <string name="release_notes_4_2_0_title_2">Opening files from outside the app using links</string>
+    <string name="release_notes_4_2_0_subtitle_2">A link that points to the app will open the app and then, resolve the file to preview it directly. Links for shared files are not included yet</string>
+    <string name="release_notes_4_2_0_title_3">"Share to" in Infinite Scale accounts allows to upload to any space</string>
+    <string name="release_notes_4_2_0_subtitle_3">Shared files from other apps can be uploaded to any space and not only the Personal one in Infinite Scale accounts</string>
+    <string name="release_notes_4_2_0_title_4">"Apply to all" for conflicts</string>
+    <string name="release_notes_4_2_0_subtitle_4">New dialog to fix all conflicts at once</string>
+    <string name="release_notes_4_2_0_title_5">Auto-refresh when a file is uploaded</string>
+    <string name="release_notes_4_2_0_subtitle_5">The file list will be now refreshed automatically while uploading to the target folder</string>
+    <string name="release_notes_4_2_0_title_6">Download log files</string>
+    <string name="release_notes_4_2_0_subtitle_6">New option to download a log file to the Downloads folder of the device</string>
     <string name="release_notes_4_2_0_title_7">Auto uploads to spaces</string>
     <string name="release_notes_4_2_0_subtitle_7">Photos and videos from camera now saved inside spaces, not only in Personal</string>
+    <string name="release_notes_4_2_0_title_8">Manage password policy in live mode in Infinite Scale accounts</string>
+    <string name="release_notes_4_2_0_subtitle_8">Password policy for public links is handled in live mode with new items in the dialog</string>
+    <string name="release_notes_4_2_0_title_9">Merge network library as module</string>
+    <string name="release_notes_4_2_0_subtitle_9">The network library has been archived and integrated into the application architecture as a module</string>
+    <string name="release_notes_4_2_0_title_10">Migration to Media3 from Exoplayer</string>
+    <string name="release_notes_4_2_0_subtitle_10">Media3 is the new home for Exoplayer, which has become a part of this library</string>
```

Comment:
Release notes should avoid technical terms like `Exoplayer` or `Media3`. Final users don't matter about library names and its integration. Just let users know which is the improvement, like `Video player improved` or something similar.